### PR TITLE
[FIX] user status Offline misnamed as Invisible in Custom Status edit dropdown menu

### LINKED ISSUE #24742: Offline misnamed as Invisible in Custom User Status edit menu
### Steps to reproduce:

1.  On the homepage, click on your avatar at the top left in the sidebar.
2. Click on **Custom Status**.
3. Click on the drop down menu button at the right side of the Status Message text field

### Expected behavior:

The 4 pre-defined user status should have the same name in both menus.
image

### Actual behavior:

**Invisible** is the same as **offline** but is misnamed as can be seen clearly.
image

### Server Setup Information:

- Version of Rocket.Chat Server:  4.5.0-develop
- Operating System: Ubuntu 20.04 running on Windows Subsystem for Linux (WSL2)
- Deployment Method: 
- Number of Running Instances: 1
- NodeJS Version: 14.18.2 - x64
- MongoDB Version: 4.4.4

### Client Setup Information

- Desktop App or Browser Version:  Google Chrome Version 98.0.4758.102 (Official Build) (64-bit)
- Operating System: Windows 11

### PR DESCRIPTION
**Pull request by Kunalvrm555**
- [x] have read the Contributing Guide - https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat doc
  - [x] I have signed the CLA - https://cla-assistant.io/RocketChat/Rocket.Chat
  - [x] Lint and unit tests pass locally with my changes
  - [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
  - [ ] I have added necessary documentation (if applicable)
  - [x] Any dependent changes have been merged and published in downstream modules

## Proposed changes (including videos or screenshots)
BEFORE :

https://user-images.githubusercontent.com/67605729/157280261-4b166dc7-755a-46bc-9d37-4f3a7f939470.mp4

AFTER :

https://user-images.githubusercontent.com/67605729/157279411-6fec52a2-8f5c-4548-947f-f3ac0c769028.mp4

## Issue(s)

Closes #24742

### CODE REVIEW

**Review by debdutdeb — DISMISSED** [review]

**Comment by debdutdeb** [comment]
@Kunalvrm555 - looks like a conflict has come to be, can you please resolve this one?

**Comment by debdutdeb** [comment]
@Kunalvrm555 - looks like a conflict has come to be, can you please resolve this one?

**Comment by debdutdeb** [comment]
@Kunalvrm555 - looks like a conflict has come to be, can you please resolve this one?

**Comment by Kunalvrm555** [comment]
> @Kunalvrm555 - looks like a conflict has come to be, can you please resolve this one?

It is showing that the conflicts are too complex to be resolved in web editor. These conflicts appeared recently, and weren't there when I opened this PR.
I don't understand, please suggest a way out @debdutdeb

File: client/components/UserStatusMenu.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		);
+
+		const statuses = [
+			['online', renderOption('online', t('Online'))],
+			['away', renderOption('away', t('Away'))],
+			['busy', renderOption('busy', t('Busy'))],
+		];
+
+		if (allowInvisibleStatus) {
+			statuses.push(['offline', renderOption('offline', t('Offline'))]);
```
Comment (Kunalvrm555) [inline client/components/UserStatusMenu.js:30]: Invisible changed to Offline

**Comment by debdutdeb** [comment]
We had a recent rewrite of this file done, where, *I may have left this unfixed* 😉 mind changing this PR to fix that? 

Ref: https://github.com/RocketChat/Rocket.Chat/pull/25265/files

**Comment by debdutdeb** [comment]
Ok, no. You're not changing the right file. It's no longer in `client` but in `apps/meteor/client`. You missed a step when resolving the conflicts. Do a simple `git mv` and it should then be fine.

**Comment by debdutdeb** [comment]
basically `git mv client/components/UserStatusMenu.tsx apps/meteor/client/components/UserStatusMenu.tsx` - amnd commit & push. (don't copy paste, I can have spelling or other mistakes)

**Comment by Kunalvrm555** [comment]
> Ok, no. You're not changing the right file. It's no longer in `client` but in `apps/meteor/client`. You missed a step when resolving the conflicts. Do a simple `git mv` and it should then be fine.

my bad, missed it.
I'l fix that. Thanks for the help 😅

**Comment by debdutdeb** [comment]
Hm it's also not up to date :/ @Kunalvrm555  - let me fix this please. I'll let you know exactly what I did so that you can follow later on.

**Comment by Kunalvrm555** [comment]
> Hm it's also not up to date :/ @Kunalvrm555 - let me fix this please. I'll let you know exactly what I did so that you can follow later on.

that'd be great. sorry for the inconvenience @debdutdeb

**Comment by debdutdeb** [comment]
No worries haha! happens. I actually didn't have to do anything else but just move the file to correct location (the command I showed above). I misread one line in the previous commit, so ignore the "not up to date" comment.

**Comment by Kunalvrm555** [comment]
> No worries haha! happens. I actually didn't have to do anything else but just move the file to correct location (the command I showed above). I misread one line in the previous commit, so ignore the "not up to date" comment.

Thank you very much 😇
Is there anything else that needs to be done now in order to merge this PR ?

**Comment by debdutdeb** [comment]
No, just waiting for all checks to turn green before approving.

**Review by debdutdeb — APPROVED** [review]

**Review by dougfabris — APPROVED** [review]

**Pull request merged**